examples/js/package.json

```diff
@@ -37,19 +37,19 @@
     "ignore": []
   },
   "dependencies": {
-    "@feathersjs/authentication": "^4.5.15",
-    "@feathersjs/authentication-local": "^4.5.15",
-    "@feathersjs/authentication-oauth": "^4.5.15",
-    "@feathersjs/configuration": "^4.5.15",
-    "@feathersjs/errors": "^4.5.15",
-    "@feathersjs/express": "^4.5.15",
-    "@feathersjs/feathers": "^4.5.15",
-    "@feathersjs/socketio": "^4.5.15",
-    "@feathersjs/transport-commons": "^4.5.15",
+    "@feathersjs/authentication": "^5.0.0-pre.19",
```

Comment:
Please update all dependencies to `pre.29` or even `pre.30`

Reply:
I have a bug in `pre.30`, there is a new dependency used for the [`typebox` package](https://github.com/feathersjs/feathers/tree/dove/packages/typebox), named `@sinclair/typebox`.
This lib is also used in `schema` package here https://github.com/feathersjs/feathers/blob/dove/packages/schema/src/json-schema.ts#L3.
As it's not added in dependency, it breaks the use of `schema` for TS. (cc @daffl maybe this is for you)

Temporary I add `@sinclair/typebox` in dev dependency to make this compile.

Reply:
Yeah, this is obviously a leftover. I created an issue here: https://github.com/feathersjs/feathers/issues/2786